katello/katello-change-hostname

```diff
@@ -170,12 +170,11 @@ def precheck
   unless @foreman_proxy_content
     STDOUT.puts "\nChecking overall health of server"
     run_cmd("hammer ping", [0], "There is a problem with the server, please check 'hammer ping'")
-  end
-
-  STDOUT.puts "\nChecking credentials"
-  hammer_cmd("capsule list") unless @foreman_proxy_content
-  unless $?.success?
-    fail_with_message("There is a problem with the credentials, please retry")
+    STDOUT.puts "\nChecking credentials"
+    hammer_cmd("capsule list")
```

Comment:
Hah, shouldn't this be

```
run_cmd("hammer -u #{@options[:username].shellescape} -p #{@options[:password].shellescape} capsule list", [0], "There is a problem with the credentials, please retry")
```

as otherwise the unless never happens?

Reply:
I would rather pass the message to hammer_cmd, and then to run_cmd, so we don't keep repeating the hammer credentials